### PR TITLE
Ft 102 export add multiple files option

### PR DESCRIPTION
**Pull request by swordEdge**

### CODE REVIEW

File: src/icons/file.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 0.5H2.5H9.5H9.70711L9.85355 0.646447L13.8536 4.64645L14 4.79289V5V15V15.5H13.5H2.5H2V15V1V0.5ZM3 1.5V14.5H13V5.5H9.5H9V5V1.5H3ZM10 2.20711L12.2929 4.5H10V2.20711Z" fill="black" fill-opacity="0.8"/>
```
Comment (six7) [inline src/icons/file.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M2 0.5H2.5H9.5H9.70711L9.85355 0.646447L13.8536 4.64645L14 4.79289V5V15V15.5H13.5H2.5H2V15V1V0.5ZM3 1.5V14.5H13V5.5H9.5H9V5V1.5H3ZM10 2.20711L12.2929 4.5H10V2.20711Z" fill="currentColor" />
```

File: src/stitches.config.ts [hunk 2]
```diff
@@ -121,6 +121,7 @@ export const stitchesInstance = createStitches({
       8: '48px',
       10: '28px',
       dropdownMaxHeight: '170px',
+      previewMaxHeight: '100px',
```
Comment (six7) [inline src/stitches.config.ts:124]: you don't have to use a token for that, it's only used once - there's no reusable value here

File: src/app/components/ExportProvider/MultiFilesExport.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    onClose();
+  }, [filesChangeset, onClose]);
+
+  return (
+    <Stack direction="column" gap={4}>
+      <Heading size="medium">Preview</Heading>
+      <Stack direction="column" gap={3} className="content scroll-container" css={{ maxHeight: '$previewMaxHeight' }}>
+        {
+          Object.keys(filesChangeset)?.map((key, index) => (
+            <Stack direction="row" gap={3} key={seed(index)}>
```
Comment (six7) [inline src/app/components/ExportProvider/MultiFilesExport.tsx:49]:
```suggestion
            <Stack direction="row" align="start" gap={3} key={seed(index)} css={{fontSize: '$xsmall'}}>
```

File: src/app/components/ExportProvider/MultiFilesExport.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      .then((content) => {
+        saveAs(content, 'tokens.zip');
+      });
+    onClose();
+  }, [filesChangeset, onClose]);
+
+  return (
+    <Stack direction="column" gap={4}>
+      <Heading size="medium">Preview</Heading>
+      <Stack direction="column" gap={3} className="content scroll-container" css={{ maxHeight: '$previewMaxHeight' }}>
```
Comment (six7) [inline src/app/components/ExportProvider/MultiFilesExport.tsx:46]:
```suggestion
      <Stack direction="column" gap={3} className="content scroll-container" css={{ maxHeight: '200px' }}>
```

**Review by six7 — CHANGES_REQUESTED** [review]
Works great! Functional/design review done, would love it if Liam could take a look from a code perspective

File: src/app/components/ExportProvider/SingleFileExport.tsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+            onCheckedChange={handleToggleExpandShadow}
+          />
+          <Label htmlFor="expandShadow">Expand Shadows</Label>
+        </Box>
+      </Box>
+      <Heading size="medium">Preview</Heading>
+      <Textarea
+        rows={10}
+        isDisabled
+        value={getFormattedTokens({
```
Comment (LiamMartens) [inline src/app/components/ExportProvider/SingleFileExport.tsx:86]: Let's `useMemo` this value; then we can also re-use it in the `Button` href

**Review by LiamMartens — CHANGES_REQUESTED** [review]

**Review by LiamMartens — APPROVED** [review]

**Pull request merged**